cvat/apps/organizations/views.py

```diff
@@ -16,6 +18,37 @@
     MembershipReadSerializer, MembershipWriteSerializer,
     OrganizationReadSerializer, OrganizationWriteSerializer)
 
+
+@extend_schema_view(retrieve=extend_schema(
```

Comment:
this is more complicated than it needs to be. following snippet generates **exactly the same** schema. some notes fyi:

- common annotations (`tags`) can be done once with `@extend_schema` on top.
- no point in overriding `responses` when it is just default behavior. in fact it's misleading since you use the automatic `request` estimation while overriding `responses`
- the version annotation is actually counter-productive because you are scoping the decorators. if someone could request `3.0`, the default behavior would run, i.e. these decorators would be simply skipped. it's advisable to only "scope-decorate" non-`DEFAULT_VERSION` behavior. 
- also the view does not make use of different versions anyway, thus the default behavior does exactly the right thing already.
- `@extend_schema_view` takes multiple args. no need for duplication

```python
@extend_schema(tags=["organizations"])
@extend_schema_view(
    retrieve=extend_schema(
        summary="Method returns details of an organization"
    ),
    list=extend_schema(
        summary="Method returns a paginated list of organizatins according to query parameters"
    ),
    update=extend_schema(
        summary="Method updates an organization by id",
    ),
    partial_update=extend_schema(
        summary="Methods does a partial update of chosen fields in an organization",
    ),
    create=extend_schema(
        summary="Method creates an organization",
    ),
    destroy=extend_schema(
        summary="Method deletes an organization",
        responses=OpenApiResponse(description="The organization has been deleted"),
    ),
)
class OrganizationViewSet(viewsets.ModelViewSet):
   ....
```